iree/tools/init_dialects.h

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This files defines a helper to trigger the registration of dialects and
+// passes to the system.
+//
+// Based on MLIR's InitAllDialects but without dialects we don't care about.
+
+#ifndef MLIR_INIT_DIALECTS_H_
```

Comment:
Should this be `IREE_TOOLS_INIT_DIALECTS_H`?

Reply:
Yeah, will change this.